components/brave_wallet/common/eth_abi_utils_unittest.cc

```diff
@@ -175,7 +175,7 @@ TEST(EthAbiUtilsTest, ExtractAddress) {
   {
     auto bytes = ToBytes(
         "000000000000000000000000c1735677a60884abbcf72295e88d47764beda282");
-    EXPECT_EQ(ExtractAddress(base::make_span(bytes)).ToHex(),
+    EXPECT_EQ(ExtractAddress(base::span(bytes)).ToHex(),
```

Comment:
nit: It seems `ExtractAddress(bytes)` would be enough

Reply:
Fixed.